source/adapters/hip/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ set(UR_HIP_INCLUDE_DIR "${UR_HIP_ROCM_DIR}/include")
 set(UR_HIP_HSA_INCLUDE_DIR "${UR_HIP_ROCM_DIR}/hsa/include")
 
 # Set HIP lib dir
-set(UR_HIP_LIB_DIR "${UR_HIP_ROCM_DIR}/hip/lib")
+set(UR_HIP_LIB_DIR "${UR_HIP_ROCM_DIR}/lib")
```

Comment:
CC @kbenzie for awareness. This change implies UR_HIP_ROCM_DIR semantic has changed. I guess this might affect the CMAKE gymnastics with regard to pointing unified-runtime/intel-llvm to the correct rocm.

Reply:
Thanks, this looks more logical to me but I'm not well versed in _finding rocm_ CMake game.

Reply:
libs found in `hip/lib` are also found in `lib` via a symlink. But logically, the path is no longer HIP but ROCM (which include ROCM). Hence the simple change here.